manifests/init.pp

```diff
@@ -0,0 +1,61 @@
+# @summary
+#   Module to install Restic and manage backups
+#
+# @param user
+#   The user to create who owns the backup files.
+#
+# @param group
+#   The group to create who owns the backup files.
+#
+# @param package_manage
+#   Enable to manage the Restic package install.
+#
+# @param package_ensure
+#   The version of Restic to install.
+#
+# @param package_name
+#   The name of the Restic package to install.
+#
+class restic (
+  String $user = 'restic',
+  String $group = 'restic',
+  Boolean $package_manage = true,
+  String $package_ensure = latest,
+  String $package_name = 'restic',
+) {
+  $homedir = "/home/${user}"
```

Comment:
Why does the user need a home in `/home`? Isn't `/var/lib/restic` or something more appropriate for automation users?

Reply:
Yes, I also had my doubts when I wrote this and then forgot it when I extracted it. Will change.